### ISSUE TITLE
[feature request] Change the style of delete buttons

### ISSUE THREAD

**Issue by BaltacMihai** [ONEARMY/community-platform#2451]
**Is your feature request related to a problem? Please describe.**
There doesn't appear to be any distinction between the importance of the buttons on the http://localhost:3000/settings page's final section, the one with "Change Email," "Change Password," and "Delete Profile," which may effect accessibility.

**Describe the solution you'd like**
A red background should be used for all buttons that have "destructive" behavior, or the ability to erase something, according to studies.

**Describe alternatives you've considered**
Another method to distinguish these buttons

**Additional context**
Several studies that support the issue: 

https://uxmovement.com/buttons/how-to-design-destructive-actions-that-prevent-data-loss/
https://uxstepbystep.com/delete-buttons-red-green/

**Comment by CYBWEBALI** [comment]
Can I work on this issue?

**Comment by BaltacMihai** [comment]
I think we need the ok from Maintainers

**Comment by iSCJT** [comment]
Can you hold on this one please, I agree with the suggestion but some of the projects colour themes already use orange/red buttons so think it needs some input from someone on the design side.  @davehakkens - thoughts?

**Comment by AlfonsoGhislieri** [comment]
Yeah this might be an issue like @iSCJT mentioned, would have to look into the color usage for the buttons across the platform. Precious plastic for example uses red as the `primary` across different buttons. Tough to bring uniformity when we have color variants for the different platforms.

**Comment by BaltacMihai** [comment]
Making the entire section different, as it was on github.

I saw that this was available in the application's profile -> admin settings section. I believe it is okay to use a red background for the sections and to provide some warning symbols to alert users when their actions may change their accounts.


![Capture](https://github.com/ONEARMY/community-platform/assets/72079422/eacfe03d-0e41-4edc-8de1-28db134cde2c)

**Comment by davehakkens** [comment]
Using the red background like for admins makes sense to me @BaltacMihai, go for it! :)

**Comment by BaltacMihai** [comment]
If @CYBWEBALI changed his mind, I can work, but he said the first time that he wanted to work.

**Comment by CYBWEBALI** [comment]
> If @CYBWEBALI changed his mind, I can work, but he said the first time that he wanted to work.

Yeah sure you can work on this.
